controller/templates/confd_settings.py

```diff
@@ -1,6 +1,6 @@
 # security keys and auth tokens
-SECRET_KEY = '{{ .deis_controller_secretKey }}'
-BUILDER_KEY = '{{ .deis_controller_builderKey }}'
+SECRET_KEY = '{{ getv "/deis/controller/secretKey" }}'
+BUILDER_KEY = '{{ getv "/deis/controller/builderKey" }}'
 
 # scheduler settings
 SCHEDULER_MODULE = '{{ or (.deis_controller_schedulerModule) "fleet" }}'
```

Comment:
I couldn't find a nice way to express default values in confd 0.7

0.5
`{{ or (.deis_controller_schedulerModule) "fleet" }}`
0.7
`{{ if exists "/deis/controller/schedulerModule" }}{{ getv "/deis/controller/schedulerModule" }}{{ else }}fleet{{ end }}`

Deis uses default values a lot, somebody has a better idea?